Add VariableDeclarationNode for binding variable (#374)
eisop/checker-framework: docs/CHANGELOG.md

**Implementation details:**

> A `VariableDeclarationNode` will be added to the CFG for the binding variable
> in a `BindingPatternTree`.

Reviewer: ```suggestion
A `VariableDeclarationNode` is now correctly added to the CFG for the binding variable
in a `BindingPatternTree`.
```
